module-services/service-cellular/CellularServiceAPI.cpp

```diff
@@ -238,6 +238,21 @@ bool CellularServiceAPI::GetAntenna(sys::Service *serv, bsp::cellular::antenna &
     return false;
 }
 
+bool CellularServiceAPI::IsCallInProgress(sys::Service *serv, bool &response)
```

Comment:
If I understand the purpose of this method correctly, it returns whether the operation was successful and the result itself is obtained via the `response` variable passed by a reference? If that's the case - it's very C-style coding pattern, IMO it'd be better to use some of the C++ mechanics designed for such cases, like `std::optional`.

Reply:
Ah, I see, let's leave it as it is then and not refactor the whole API ;)